backend/src/utils/schedule/generate_schedule.py

```diff
@@ -0,0 +1,55 @@
+from pyschedule import Scenario, solvers, plotters, alt
```

Comment:
@achcello - how should we have this file version controlled since its deployment will be controlled via AWS?

Reply:
We have an H4I AWS, I can move it over there. I'd add to one of the README's explaining setup for that.

Reply:
Sorry misunderstood. But yes, if you could write a README for the setup details we can move over to that ASAP, not a blocker though. In terms of placement for this, I don't think it makes sense to keep it in here. I'd actually put it in another top-level directory in this repo (e.g. `scheduler`). Then we can also easily add this README there with setup instructions for this part, since it is a separate deployment.